Vagrantfile

```diff
@@ -13,10 +13,16 @@ apt-get update
 apt-get install -y tor deb.torproject.org-keyring
 
 # Setup for sniffer subsystem
-apt-get install -y build-essential libdumbnet-dev libpcap-dev libgeoip-dev libffi-dev python-dev python-pip
+apt-get install -y build-essential libdumbnet-dev libpcap-dev libgeoip-dev libffi-dev python-dev python-pip libssl-dev
 cd /data/ooni-probe
 python setup.py install
 
+
+echo "To run ooniprobe:"
+echo "1. vagrant ssh probe"
+echo "2. ooniprobe-agent start"
+echo "3. connect to the web UI on your host machine at http://localhost:8042/"
```

Comment:
Since this is running on a Vagrant, there should be no browser. Does `ooniprobe-agent` listen on default only on the localhost or on any interface? Probably it should listen on localhost by default and on any interface only if enabled by the user. In any case, I guess this instructions above require attention.